dashboard/sl_queries.py

```diff
@@ -20,7 +20,7 @@ def get_connection() -> connect:
                    database=ENV["DB_NAME"])
 
 
-@st.cache_data
+@st.cache_data(ttl=900)
```

Comment:
The following functions could all be in the same place, so you won't have to duplicate this.

Reply:
Nevermind, I just realised this is a decorator.